lib/server.js

```diff
@@ -41,6 +42,11 @@ function startServer (opts) {
     return data
   }
 
+  function resolveTopojson (url) {
+    var data = fs.readFileSync(path.join(topojsonPath, path.basename(url))).toString()
```

Comment:
(The above is readsync + etc because of the json validation step, but I think returning a stream should work and is (infinitesimally) faster)